src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+    </tr>
+    <tr>
+        <td><inlineCode>requiredPermissions</inlineCode></td>
+        <td><inlineCode>PermissionsDefinition</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Indicates the various permissions this plugin requires. These permissions are required before accessing certain API surfaces. Without these permissions in the manifest, API access may fail or throw an error. Some permissions require user consent.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no permissions)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>addon</inlineCode></td>
+        <td><inlineCode>object</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Addon definitions for hybrid plugins. See the hybrid plugin documentation for more information. A UXP Hybrid plugin is a UXP plugin that can access the power of C++ native libraries. This currently isn't supported in InDesign.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no addons)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>featureFlags</inlineCode></td>
+        <td><inlineCode>FeatureFlags</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>
+                A set of feature flags that can be used to enable or disable certain features of the plugin. These flags are used to gate features that are not yet ready for general availability. 
+            </p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no additional feature flags)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### StringsDefinition
+
+Represents a set of strings used to localize the plugin name and other user-facing strings.
+
+`StringsDefinition` keys can be used anywhere where `LocalizedString` is supported.
+
+**Example**
+
+Your manifest.json file might look like this:
+
+```json
+{
+	"name": "my-plugin",
+	"strings": {
+		"my-plugin": {
+			"default": "My Plugin",
+			"de": "Mein Plugin"
+		}
+	}
+}
+```
+
+### LocalizedString
+
+Represents a localized string. The key is the locale, and the value is the translated string.
+
+**Example**
+
+```json
+{
+	"default": "Hello",
+	"en": "Hello",
+	"de": "Hallo"
+}
+```
+
+#### Properties
+
+<table>
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>default</inlineCode> *</td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The default string used when no translation is available for the current locale.</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### IconDefinition
+
+<p>Represents an icon used by the plugin. The icon may be used in the plugin list, toolbar, or other places.</p>
```

Comment:
Replace "Represents an icon used by the plugin" with "Represents an icon used by the plugin or specific entry point"